target/linux/mediatek/image/mt7623.mk

```diff
@@ -5,6 +5,8 @@ define Device/bpi_bananapi-r2
   DEVICE_MODEL := Banana Pi R2
   DEVICE_DTS := mt7623n-bananapi-bpi-r2
   SUPPORTED_DEVICES := bananapi,bpi-r2
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb-ohci kmod-usb2 kmod-usb3 \
+                     kmod-ata-core kmod-ata-ahci-mtk
```

Comment:
kmod-usb-core and kmod-ata-core don't need to be specified here, as they will be selected by the other packages anyway.